modules/opensearch/main.tf

```diff
@@ -22,7 +22,7 @@ resource "aws_ssm_parameter" "es_user" {
 resource "aws_ssm_parameter" "es_password" {
   name      = "/${var.name}/es/master/password"
   type      = "SecureString"
-  value     = random_password.es.result
+  value     = nonsensitive(data.aws_secretsmanager_secret_version.os_password.secret_string)
```

Comment:
We need to replace references to `es/master/password` SSM parameter with the new Secrets Manager key ARN.
We shouldn't decrypt the secrets and use them to set attributes like here. Terraform keeps this sensitive value in the remote state which is not safe.